spec-main/webview-spec.ts

```diff
@@ -435,6 +435,11 @@ describe('<webview> tag', function () {
     };
 
     afterEach(closeAllWindows);
+    afterEach(async () => {
+      // The leaving animation is un-observable but can interfere with future tests
```

Comment:
> un-observable

can we make it observable? or perhaps poll?

Reply:
Unfortunately not, the OS transitions the "fullscreen" property before the animation finishes, and if another thing tries to go fullscreen while something else is finishing the animation it simply won't ever enter fullscreen (it doesn't queue it)